ui/app/src/Utils/Filter.elm

```diff
@@ -48,7 +48,7 @@ generateQueryString { receiver, showSilenced, text, group } =
     let
         -- TODO: Re-add receiver once it is parsed on the server side.
         parts =
-            [ ( "silenced", Maybe.map (toString >> String.toLower) showSilenced )
+            [ ( "silenced", Maybe.withDefault False showSilenced |> toString |> String.toLower |> Just )
```

Comment:
Is there a reason to force this to be `Just True` or `Just False`? The server code checks to see if the query string exists, so I don't think there's anything wrong with this being `Nothing` and just having it drop out from the api call.